lib/cryptosuites/suites.go

```diff
@@ -202,6 +205,7 @@ var (
 		GitClient:           Ed25519,
 		AWSRACATLS:          ECDSAP256,
 		BoundKeypairJoining: Ed25519,
+		BoundKeypairCAJWT:   Ed25519,
```

Comment:
New CAs should use `ECDSAP256` for legacy to be compatible with KMS/HSM.

https://github.com/gravitational/teleport/blob/master/rfd/0136-modern-signature-algorithms.md#legacy-suite

I learned it the hard way and had to rollback https://github.com/gravitational/teleport/pull/53444

Reply:
Yikes, good catch! I've swapped the algorithm like you suggested.

Reply:
...and I've also added a note about this with a link to the RFD to hopefully avoid this in the future.